plugins/BEdita/Core/tests/TestCase/Model/Behavior/UniqueNameBehaviorTest.php

```diff
@@ -49,6 +49,18 @@ class UniqueNameBehaviorTest extends TestCase
     public function setUp()
     {
         parent::setUp();
+
```

Comment:
Probably a `ModelTestCase` base class with `setUp` and `tearDown` and `LoggedUser` methods could be handy, to avoid repetitions and potential problems on model tests. 

Not necessary now, could be a in a next refactor...